### ISSUE TITLE
Bad handling of empty theme parameter in config.yml

### ISSUE THREAD

**Issue by hlef** [hlef/juliet#3]
config.yml files with empty theme parameter are not rejected currently. As a result an empty "assets" folder is built.

e.g. the default config

```
# useful when your website isn't located at the root of your server
baseurl: ""

# the directory in themes/ that contain your theme
# WARNING: No theme under themes/ yet ! More informations in the documentation:
# https://github.com/hlef/juliet
theme: ""
```

This configuration should not be buildable (it doesn't make sense, there's nothing to build). Instead a meaningful error message should be displayed to the user.

**Comment by hlef** [comment]
There is not validation of the configuration at the moment. This is the problem.

**Comment by hlef** [comment]
Fixed in 6873d2860a5f5634d95ad5f8efb405f5dc11b54c.
